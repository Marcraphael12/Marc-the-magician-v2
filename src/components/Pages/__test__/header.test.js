import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { BrowserRouter } from 'react-router-dom';
import Header from '../header';

describe('Header page of the App', () => {
  test('Builds the snapchot of the Header component safely', () => {
    const rend = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>
    );
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Header component', () => {
    render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>
    );
  })
});