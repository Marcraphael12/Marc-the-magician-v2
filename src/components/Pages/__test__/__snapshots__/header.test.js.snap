// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header page of the App Builds the snapchot of the Header component safely 1`] = `
<div
  className="Header"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
